extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -873,12 +875,35 @@ export class AmpA4A extends AMP.BaseElement {
     });
   }
 
+  /** @override **/
+  attemptChangeSize(newHeight, newWidth) {
+    // Store original size of slot in order to allow re-expansion on
+    // unlayoutCallback so that it is reverted to original size in case
+    // of resumeCallback.
+    this.originalSlotSize_ = this.originalSlotSize_ || this.getLayoutBox();
+    return super.attemptChangeSize(newHeight, newWidth).catch(() => {});
+  }
+
   /** @override  */
   unlayoutCallback() {
     // Increment promiseId to cause any pending promise to cancel.
     this.promiseId_++;
     this.protectedEmitLifecycleEvent_('adSlotCleared');
     this.uiHandler.setDisplayState(AdDisplayState.NOT_LAID_OUT);
+    if (this.originalSlotSize_) {
+      super.attemptChangeSize(
```

Comment:
That'd be dangerous since unlayout is not necessary due to whole document being backgrounded. I think it'd be better to use "attempt", but ensure that in `Resources` "attempt" is equivalent to "certain change" when `viewer.isVisible() == false`. I think it should be the case now, but always good to check.